modules/peers.js

```diff
@@ -545,7 +626,7 @@ Peers.prototype.onPeersReady = function () {
 		});
 	}
 	// Loop in 10sec intervals (5sec + 5sec connect timeout from pingPeer)
-	jobsQueue.register('peersDiscoveryAndUpdate', peersDiscoveryAndUpdate, 5000);
+	jobsQueue.register('peersDiscoveryAndUpdate', peersDiscoveryAndUpdate, 10000);
```

Comment:
So now peers state is mainteaned in real time, so we don't need to recheck peers every 10 sec, can be moved to something like 30 or 60s.